Distributor documentation pr template (#26)
MewsSystems/gitbook-distributor-guide: distributor-api-v1/operations.md

| :--- | :--- | :--- | :--- |
| `Cities` | array of [City](operations.md#city) | required | Cities supported by hotel. |
| `CityId` | string | required | ID of default city. |

> | `RequiredFields` | array of [RequiredField](operations.md#requiredfield) | required | Fields that enterprise sets up as required. |

Reviewer: @dobras 
```suggestion
| `RequiredFields` | array of [RequiredField](operations.md#requiredfield) | required | Fields that are set up as required in distributor configuration. |
```

| `Theme` | string | optional | TBC |
| `VoucherCode` | string | optional | TBC |

> #### Requiredfield

Reviewer: @dobras This is inconsistent with the name of the type in the table, should be `RequiredField`.
